src-built-in/components/linker/src/linker.tsx

```diff
@@ -0,0 +1,75 @@
+/*!
+* Copyright 2017 by ChartIQ, Inc.
+* All rights reserved.
+*/
+import * as React from "react";
+import { useEffect } from "react";
+import { Provider, useSelector, useDispatch } from 'react-redux';
+import * as ReactDOM from "react-dom";
+
+import "./css/linkerWindow.css";
+import "../../../../assets/css/font-finance.css";
+import "../../../../assets/css/finsemble.css";
+import { RootState } from '../../UIAPI/types';
+import store from '../../UIAPI/store';
+import * as linkerActions from "../../UIAPI/actions/linkerActions";
+
+declare var FSBL: any;
```

Comment:
I think we're doing this in many places. Might be a good idea to move them to a ts declaration file to avoid repeating.